snippets/base/models.py

```diff
@@ -392,7 +392,8 @@ class Snippet(CachingMixin, SnippetBaseModel):
 
     campaign = models.CharField(
         max_length=255, blank=True, default='',
-        help_text='Optional campaign name. Will be added in the stats ping.')
+        help_text=('Optional campaign name. Will be added in the stats ping. '
+                   'If be used for snippet blocking if set.'))
```

Comment:
I think there may be a typo in "If be used for snippet blocking if set."

Not really sure what you were intending though. "To be used for snippet blocking if set."? maybe.